_posts/2023-11-23-geospatial-techno.md

```diff
@@ -0,0 +1,76 @@
+GeoServer can be installed on different operating systems, since it's a Java based application. You can run it on any kind of operating system for which exists a Java virtual machine. GeoServer’s speed depends a lot on the chosen Java Runtime Environment (JRE). The latest versions of GeoServer are tested with both OracleJRE and OpenJDK. These versions are:
+- Java 17 for GeoServer 2.23 and above
+- Java 11 for GeoServer 2.15 and above
+- Java 8 for GeoServer 2.9 to GeoServer 2.22
+- Java 7 for GeoServer 2.6 to GeoServer 2.8
+- Java 6 for GeoServer 2.3 to GeoServer 2.5
+- Java 5 for GeoServer 2.2 and earlier
+
+There are many ways to install GeoServer on your system. This tutorial will use the Oracle JRE8 also known as JRE 1.8 and discuss the two most commonly used installation methods for windows.
+- Windows Installer
+- Web Archive
+
+### Windows installer
+The Windows installer provides an easy way to set up GeoServer on your system, as it requires no configuration files to be edited or command line settings.
+
+#### Installation
+- Navigate to Oracle.com and download JRE 8 from the Products tab. Consider the operating system architecture and memory requirements when selecting a JRE installer. 32-bit Java version is restricted to 2 GB memory, while the 64-bit version is recommended for optimal server memory. Utilizing JAI with the 32-bit JRE can enhance performance for WMS output generation and raster operations.
+- Install JRE by following the default settings and successfully complete the installation.
+- Navigate to the GeoServer.org and download the desired version of GeoServer.
+- Launch the GeoServer installer and agree to the license.
+- Enter the path to the JRE installation and proceed with the installation. The installer will attempt to automatically populate this box with a JRE if it is found, but otherwise you will have to enter this path manually.
+- Provide necessary details like the GeoServer data directory, administration credentials, and port configuration.
+- Review the selections, install GeoServer, and start it either manually or as a service.
+- Finally, navigate to localhost:8080/geoserver (or wherever you installed GeoServer) to access the GeoServer Web administration interface.
+
+#### Uninstallation
+GeoServer can be uninstalled in two ways:
+- By running the uninstall.exe file in the directory where GeoServer was installed
+- By standard Windows program removal
+
+### Web archive
+GeoServer is packaged as a standalone servlet for use with existing application servers such as Apache Tomcat, GlassFish, Jetty and JBoss. It has been mostly tested using Tomcat, and so is the recommended application server. There are reasons for installing it such as it is widely used, well-documented, and relatively simple to configure. GeoServer requires a newer version of Tomcat (7.0.65 or later) that implements Servlet 3 and annotation processing. Other application servers have been known to work, but are not guaranteed.
```

Comment:
This sentence does not make sense to me:

1. It is packaged as a web-archive
2. It is not standalone anything; as a web-archive requires an application server to run :)

```
GeoServer is packaged as a web-archive (WAR) for use with an application server such as Apache Tomcat or Jetty.
```

Glassfish and JBoss are not used so much anymore.

Reply:
You are right